pkg/controller/machineautoscaler/machinetarget.go

```diff
@@ -33,50 +33,50 @@ func (mt *MachineTarget) NeedsUpdate(min, max int) bool {
 	return minDiff || maxDiff
 }
 
-// SetLimits sets the target's min and max labels.
+// SetLimits sets the target's min and max annotations.
 func (mt *MachineTarget) SetLimits(min, max int) {
-	labels := mt.GetLabels()
+	annotations := mt.GetAnnotations()
 
-	if labels == nil {
-		labels = make(map[string]string)
+	if annotations == nil {
+		annotations = make(map[string]string)
 	}
 
-	labels[minSizeLabel] = strconv.Itoa(min)
-	labels[maxSizeLabel] = strconv.Itoa(max)
+	annotations[minSizeAnnotation] = strconv.Itoa(min)
+	annotations[maxSizeAnnotation] = strconv.Itoa(max)
 
-	mt.SetLabels(labels)
+	mt.SetAnnotations(annotations)
 }
 
-// RemoveLimits removes the target's min and max labels.
+// RemoveLimits removes the target's min and max annotations.
 func (mt *MachineTarget) RemoveLimits() {
-	labels := mt.GetLabels()
+	annotations := mt.GetAnnotations()
 
-	delete(labels, minSizeLabel)
-	delete(labels, maxSizeLabel)
+	delete(annotations, minSizeAnnotation)
+	delete(annotations, maxSizeAnnotation)
 
-	mt.SetLabels(labels)
+	mt.SetAnnotations(annotations)
 }
 
 // GetLimits returns the target's min and max limits.  An error may be
-// returned if the label's contents could not be parsed as integers.
+// returned if the annotations's contents could not be parsed as ints.
 func (mt *MachineTarget) GetLimits() (min, max int, err error) {
-	labels := mt.GetLabels()
+	annotations := mt.GetAnnotations()
 
-	minString, minOK := labels[minSizeLabel]
-	maxString, maxOK := labels[maxSizeLabel]
+	minString, minOK := annotations[minSizeAnnotation]
+	maxString, maxOK := annotations[maxSizeAnnotation]
 
 	if !minOK || !maxOK {
-		return 0, 0, ErrTargetMissingLabels
+		return 0, 0, ErrTargetMissingAnnotations
 	}
 
 	min, err = strconv.Atoi(minString)
 	if err != nil {
-		return 0, 0, fmt.Errorf("bad min label: %s", minString)
+		return 0, 0, fmt.Errorf("bad min annotation: %s", minString)
```

Comment:
Perhaps we could convey the annotation key, might make it easier for someone to debug as there would be something concrete to grep for:
 
    return 0, 0, fmt.Errorf("bad min annotation for %q: %s", minSizeAnnotation, minString)

or something like that.